src/pages/game.jsx

```diff
@@ -1,17 +1,156 @@
-import React from 'react';
+import React, { useReducer } from 'react';
 import Layout from '../components/Layout';
 import Picture from '../components/Picture';
 import Sentence from '../components/Sentence';
 import Guess from '../components/Guess';
 import LetterSelection from '../components/LetterSelection';
 
+const types = {
+  nextPage: 'nextPage',
+  prevPage: 'prevPage',
+  checkWord: 'checkWord',
+};
+
+const reducer = (state, action) => {
+  switch (action.type) {
+    case types.nextPage: // increments counter in gameState
+      if (
```

Comment:
What's DDK? Also, what's rough the read? The comment itself or the reducer switch case? And add comments for what?
(I am still retroactively adding comments as I see fit, but I want to know what you mean here)

Reply:
Doug Doug Kaung like our Slack Workspace. You know what ... I'm sorry, it's the two lines below this. I should have put this comment on line 19 instead of above it. My mistake.